src/wp-includes/pluggable.php

```diff
@@ -1917,11 +1931,17 @@ function wp_notify_postauthor( $comment_id, $deprecated = null ) {
 					break;
 			}
 
-			$notify_message .= get_permalink( $comment->comment_post_ID ) . "#comments\r\n\r\n";
 			/* translators: %s: Comment URL. */
-			$notify_message .= sprintf( __( 'Permalink: %s' ), get_comment_link( $comment ) ) . "\r\n";
+			if ( 'note' === $comment->comment_type ) {
+				$notify_message .= admin_url( "post.php?post={$comment->comment_post_ID}" ) . "\r\n";
```

Comment:
The `action=edit` parameter is missing. It might be better to use the `get_edit_post_link()` function.
```suggestion
				$notify_message .= get_edit_post_link( $comment->comment_post_ID, 'url' ) . "\r\n";
```

Reply:
Nice one!